app/presenters/search/search.presenter.js

```diff
@@ -9,11 +9,22 @@ const ContactModel = require('../../models/contact.model.js')
 const { formatLongDate, formatReturnLogStatus } = require('../base.presenter.js')
 const { today } = require('../../lib/general.lib.js')
 
+const filterItems = [
```

Comment:
I'll pin my note here, but it applies to `_filterItems()` as well.

We've almost doubled the lines of code and added more complexity with `filter()` and `map()` to get the same result.

I'm very much a proponent of 'dumb' code. The previous iteration required little to no thought. So I'm not seeing the benefit of this alternate implementation, but feel free to tell me why I'm wrong!

Reply:
Agreed - I will revert this.

(In my head, what was effectively two lines of code in `_filterItems()` was short and sweet, but as it's actually spread out over 11 lines it's considerably less so!)